documentation/asset-management/topics/custom-properties.rst

```diff
@@ -0,0 +1,20 @@
+.. include:: /asset-management/media.rst
+.. _Custom-properties-doc:
+
+*****************
+Custom properties
+*****************
+Asset custom properties allow users to extend the range of data that is stored in Hyperview. Use cases include but are not limited to:
+
+- Add data to link assets to a different system, such as financial systems
+- Add data to give users of the application more context about the asset, such as project information
+- Anything else users may need to add to improve efficiency and communication
+
+Custom properties can be managed by navigating to the :ref:`Settings -> Custom properties<Managing-custom-properties-doc>` page.
```

Comment:
should be "Settings -> Custom Properties" (Properties with a capital P)

Reply:
Also we use "→" instead of "->"

Reply:
Addressed, Thanks!